### ISSUE TITLE
Update tctl sso examples to obfuscate secrets.

### ISSUE THREAD

**Issue by alexfornuto** [gravitational/teleport#29278]
Since the use of `--secret` is now in several docs, I'll make a new issue that once this PR (#29087) is merged, we should update all the SSO guides to use this method.

> I don't think so, but we could have the user put the secret in a file and use `--secret $(cat client-secret.txt)` or something.

_Originally posted by @alexfornuto in https://github.com/gravitational/teleport/pull/29087#discussion_r1267306892_
